app/models/conference.js

```diff
@@ -1,15 +1,17 @@
 import attr from 'ember-data/attr';
-import Collection from './collection';
+//import Collection from './collection';
+import Model from 'ember-data/model';
 
-export default Collection.extend({
-  city: attr('string'),
-  state: attr('string'),
-  country: attr('string'),
-  start: attr('date'),
-  end: attr('date'),
-  submissionStart: attr('date'),
-  submissionEnd: attr('date'),
-  description: attr('string'),
-  siteUrl: attr('string'),
-  logoUrl: attr('string'),
+export default Model.extend({
+    title: attr('string'),
+    city: attr('string'),
+    state: attr('string'),
+    country: attr('string'),
+    start: attr('isodate', { defaultValue : (new Date()).toISOString() }),
+    end: attr('isodate', { defaultValue : (new Date()).toISOString() }),
+    submissionstart: attr('isodate', { defaultValue : (new Date()).toISOString() }),
```

Comment:
Please consider using camel case for variables in JS. Why did you change to all lower case?